src/Round.js

```diff
@@ -1,35 +1,32 @@
 const Turn = require("./Turn");
 
 class Round {
-    constructor(deck) {
-        this.playingDeck = deck.cards;
-        this.turn;
-        this.turnsCount = 0;
-        this.currentCard = deck.cards[this.turnsCount];
-
-        this.incorrectGuesses = [];
-        this.correctGuesses = [];
-    }
+  constructor(deck) {
+    this.playingDeck = deck.cards;
+    this.turn;
+    this.turnsCount = 0;
+    this.currentCard = deck.cards[0];
+    this.incorrectGuesses = [];
+    this.correctGuesses = [];
+  }
 
-    returnCurrentCard() {
-        return this.currentCard;
-    }
+  returnCurrentCard() {
+    return this.currentCard;
+  }
 
-    takeTurn(guess) {
-        this.turnsCount++;
-        let currentPlayingCard = this.playingDeck;
+  takeTurn(guess) {
+    let currentPlayingCard = this.playingDeck;
+    this.turn = new Turn(guess, this.currentCard);
+    let answer = this.turn.evaluateGuess()
         
+    if(!answer) {
+        this.incorrectGuesses.push(this.currentCard.id);
+    } else {
+        this.correctGuesses.push(this.currentCard.id);
+    }
+        this.turnsCount += 1;
```

Comment:
I appreciated seeing how the game bug was fixed when I moved the incrementor down.